documentation/tutorials/creating_synthetic_tokens_from_truffle_console.md

```diff
@@ -0,0 +1,151 @@
+# Creating Synthetic Tokens from Truffle Console
+
+This tutorial will show you how to create synthetic tokens from the command line using UMA’s synthetic token template. Before beginning this tutorial, please make sure your environment is set up correctly by following the instructions in the [Prerequisites](prerequisites.md). After completing this section, you should: 
+* Have the protocol repo cloned.
+* Be running an instance of Ganache on port 9545.
+* Have run truffle migrations for the contracts in `core/`.
+
+Below, we’ll discuss how to create and manage a token sponsor position via the command line on a local testnet. 
+UMA also has a command-line interface (CLI) tool that bundles and renames some of these steps. 
+These tutorials discuss how to create and manage a token sponsor position using this CLI tool. 
+<!-- TODO: links to separate page -->
+
+* Begin [here](#parameterize-and-deploy-a-contract) if you are creating a new type of synthetic token on a local testnet.
```

Comment:
@rcai1 I know we've discussed this breakage, but could you make an issue under the new documentation milestone I created? https://github.com/UMAprotocol/protocol/milestone/12

Reply:
Done - please see https://github.com/UMAprotocol/protocol/issues/1055